website/src/components/NetworkPage.tsx

```diff
@@ -0,0 +1,137 @@
+import { NetworkType } from '@pinax/graph-networks-registry'
+import Head from 'next/head'
+import { useRouter } from 'next/router'
+import { useData } from 'nextra/hooks'
+import { memo } from 'react'
+
+import { ExperimentalCopyButton, ExperimentalDescriptionList, ExperimentalLink, Grid, Skeleton } from '@edgeandnode/gds'
+import { NetworkIcon } from '@edgeandnode/go'
+
+import NetworkDetailsPage from '@/components/NetworkDetailsPage'
+import { useI18n } from '@/i18n'
+import { getIconVariant, shouldShowSkeleton } from '@/utils/networkUtils'
+
+type CAIP2Id = `${string}:${string | number}`
+
+interface Network {
+  id: string
+  fullName: string
+  shortName?: string
+  networkType: NetworkType
+  graphNode?: {
+    protocol: string
+  }
+  caip2Id?: CAIP2Id
+  nativeToken?: string
+  docsUrl?: string
+}
+
+interface NetworkPageProps {
+  network?: Network
+}
+
+export const NetworkPage = memo(({ network }: NetworkPageProps) => {
+  const data = useData()
+  const contextNetwork = data?.ssg?.network || data?.network
+  const networkData: Network = network ||
+    contextNetwork || {
+      id: '',
+      fullName: '',
+      networkType: NetworkType.Mainnet,
+    }
+  const { t } = useI18n()
+  const router = useRouter()
+
+  return (
+    <>
+      <Head>
+        <title>{networkData.fullName}</title>
+      </Head>
+      <div className="col-[container]">
+        <div className="mb-5 mt-12 flex flex-col gap-3">
+          {networkData.caip2Id ? (
+            <div className="flex-shrink-0">
+              {shouldShowSkeleton(networkData.id) ? (
+                <Skeleton borderRadius="FULL" height="40px" width="40px" />
+              ) : (
+                <NetworkIcon
+                  variant={getIconVariant(networkData.id)}
+                  caip2Id={networkData.caip2Id as any}
+                  size={10}
+                  className="h-10 w-10"
+                />
+              )}
+            </div>
+          ) : null}
+          <h2 className="leading-tight mt-0 text-24 text-white">{networkData.fullName}</h2>
+        </div>
+
+        <Grid className="gap-4">
```

Comment:
This is an old component, we should really mark it as deprecated in GDS (I guess it will be once I get to [this task](https://linear.app/thegraph/issue/GDS-298/update-file-structure-and-deprecatepromote-components)). The new way is to use Tailwind's `grid` class on a `div` (or any element).

Reply:
(Same for `Flex` and `Text`)